python/tvm/relay/frontend/paddlepaddle.py

```diff
@@ -502,7 +502,9 @@ def convert_dropout(g, op, block):
     """Operator converter for dropout."""
 
     x = g.get_node(op.input("X")[0])
-    g.add_node(op.output("Out")[0], x)
+    dropout_prob = op.attr("dropout_prob")
+    out = _op.nn.dropout(x, dropout_prob)
+    g.add_node(op.output("Out")[0], out)
```

Comment:
There's two mode in dropout, refer https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/nn/Dropout_cn.html#dropout for more details

Reply:
Done.